moderncv-template.tex

```diff
@@ -0,0 +1,257 @@
+%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% "ModernCV" CV and Cover Letter
+% LaTeX Template
+% Version 1.1 (9/12/12)
+%
+% This template has been downloaded from:
+% http://www.LaTeXTemplates.com
+%
+% Original author:
+% Xavier Danaux (xdanaux@gmail.com)
+%
+% License:
+% CC BY-NC-SA 3.0 (http://creativecommons.org/licenses/by-nc-sa/3.0/)

```

Comment:
If you want this in the repo go with the project's [LPPL-1.3c](https://spdx.org/licenses/LPPL-1.3c.html).

Reply:
I have updated the license to the project's LPPL-1.3c. Hope this works!